android/common/src/main/java/org/mozilla/firefox/qt/common/Utils.kt

```diff
@@ -43,7 +43,12 @@ object Utils {
     }
     @SuppressLint("NewApi")
     @JvmStatic
-    fun initializeGlean(ctx: Context, isTelemetryEnabled: Boolean, channel: String) {
+    fun initializeGlean(ctx: Context, isTelemetryEnabled: Boolean, channel: String,
+        isGleanDebugTagActive: Boolean) {
+        if (isGleanDebugTagActive) {
+            Log.i("VPNUtils", "Setting Glean debug tag.")
+            Glean.setDebugViewTag("VPNTest")
```

Comment:
I am not a big fan of pinning this to a single value. Since we have to pass a value anyways to the daemon, why not pass a string instead of a boolean?